documentation/asciidoc/computers/raspberry-pi/bcm2711-bootloader.adoc

```diff
@@ -280,6 +284,69 @@ Previously this property was only checked by the `rpi-eeprom-update` script. How
 
 Default: `0`
 
+[[HTTP_HOST]]
+==== HTTP_HOST
+
+If an HTTP download is initiated, `boot.img` and `boot.sig` are downloaded from this server.
```

Comment:
I still think this "HTTP download" language is confusing. _If_ I'm understanding things correctly (?) "HTTP downloads" / "HTTP boot" / "HTTP boot over ethernet" are exactly the same thing as "network install", just triggered from the `BOOT_ORDER` setting, instead of being triggered by pressing SHIFT at power-on?
So perhaps it makes sense to just refer to "network install" in all circumstances?

Reply:
I think I was trying to draw a distinction between network install and http boot as you can disable network install but still use HTTP to load an image (which might not be the network install image). So I've used "HTTP download" here to refer to the download mechanism behind network install and HTTP boot...

Reply:
I've changed it to "If network install or HTTP boot is initiated"